packages/dd-trace/src/profiling/config.js

```diff
@@ -66,6 +71,18 @@ class Config {
     this.endpointCollection = endpointCollection
     this.pprofPrefix = pprofPrefix
 
+    const oomMonitoringEnabled = coalesce(options.oomMonitoring,
+      DD_PROFILING_EXPERIMENTAL_OOM_MONITORING_ENABLED, false)
+    const heapLimitExtensionSize = coalesce(options.oomHeapLimitExtensionSize,
+      Number(DD_PROFILING_EXPERIMENTAL_OOM_HEAP_LIMIT_EXTENSION_SIZE), 0)
+    const maxHeapExtensionCount = coalesce(options.oomMaxHeapExtensionCount,
+      Number(DD_PROFILING_EXPERIMENTAL_OOM_MAX_HEAP_EXTENSION_COUNT), 0)
+    const exportStrategies = ensureOOMExportStrategies(coalesce(options.oomExportStrategies,
+      DD_PROFILING_EXPERIMENTAL_OOM_EXPORT_STRATEGIES), this)
+    this.oomMonitoring = { enabled: oomMonitoringEnabled,
+      heapLimitExtensionSize,
+      maxHeapExtensionCount,
+      exportStrategies }
```

Comment:
Generally we would structure a multiline object like this:

```suggestion
    this.oomMonitoring = {
      enabled: oomMonitoringEnabled,
      heapLimitExtensionSize,
      maxHeapExtensionCount,
      exportStrategies
    }
```

Reply:
Is there a tool that does that automatically ?
Linter sometimes complains about the formatting but not in this case.

Reply:
Hmm...I would have expected eslint to capture that. Generally `eslint --fix` would fix things like this for us, but it won't do anything if it's not seeing a problem. 🤔